pydra/engine/tests/test_shelltask.py

```diff
@@ -627,6 +627,70 @@ def gather_output(keyname, output_dir):
     assert all([file.exists for file in res.output.newfile])
 
 
+@pytest.mark.parametrize("plugin", Plugins)
+def test_shell_cmd_outputspec_5_nosubm(plugin):
+    """
+        providing output name using input_spec,
+        specify in metadata output=True and provide name_tamplate
+        no submitter
+    """
+    cmd = "touch"
+    args = "newfile_tmp.txt"
+
+    my_input_spec = SpecInfo(
+        name="Input",
+        fields=[
+            (
+                "out1",
+                File,
```

Comment:
this would be a string on the input side, but a File on the output side.

Reply:
ok, makes sense